src/libs/Navigation/AppNavigator/AuthScreens.js

```diff
@@ -183,6 +185,19 @@ class AuthScreens extends React.Component {
             // when displaying a modal. This allows us to dismiss by clicking outside on web / large screens.
             isModal: true,
         };
+        const fullscreenModalScreenOptions = {
+            ...modalScreenOptions,
+            cardStyle: {...styles.fullscreenCard, ...styles.bgTransparent},
+            cardStyleInterpolator: props => fullscreenModalcardStyleInterpolator(
+                this.props.isSmallScreenWidth,
+                props,
+            ),
+            cardOverlayEnabled: false,
+
+            // We don't want the Pressable overlay for fullscreen modal which closes the modal on click
+            isModal: false,
```

Comment:
Do we need to explicitly pass `false` here? I think it will work OK with it left out

https://github.com/Expensify/Expensify.cash/blob/2863305372b498aac8a5007cb73738115c303f2d/src/libs/Navigation/AppNavigator/createCustomModalStackNavigator.js#L16-L16

Reply:
I am extending the modalScreenOptions at the top in fullscreenModalScreenOptions thus it needs to be overridden.

Reply:
Ah ok, let's instead add it to where it is needed and make the default thing we are extending not have it?